docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md

```diff
@@ -477,7 +477,7 @@ The steps in this section add properties and helper methods to the form class th
                 UpdateStatus("Workflow Canceled.")
             Else
                 Dim Turns As Integer = Convert.ToInt32(e.Outputs("Turns"))
```

Comment:
```suggestion
                Dim turns As Integer = Convert.ToInt32(e.Outputs("Turns"))
```

Reply:
@Youssef1313 I don't disagree, but I'd like to keep the scope of these changes to formatting specifically so as not to muddy the PR. I think camel-casing local variables is a fine PR idea; I just don't want to mix-and-match here.